db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170319214240) do
+ActiveRecord::Schema.define(version: 20170613071826) do
```

Comment:
I was actually using the multiple migration files to generate the files! but I should've used the schema.db instead.
Done: https://github.com/mena-devs/tilde/pull/7/commits/2d82bd3f17b522e1e6f87e5eadf2bfe57551e196